Gems/DebugDraw/Assets/Shaders/SphereIntersection.shader

```diff
@@ -0,0 +1,43 @@
+{
+    "Source" : "SphereIntersection.azsl",
+    "DrawList" : "RayTracing",
+
+    "AddBuildArguments": 
+    {
+        "dxc": ["-fspv-target-env=vulkan1.2"]
+    },
+
+    "ProgramSettings":
+    {
+        "EntryPoints":
+        [
+            {
+                "name": "Intersection",
+                "type": "RayTracing"
+            }
+        ]
+    },
+
+    "DisabledRHIBackends": ["metal"],
+
+    "Supervariants":
+    [
+        {
+            "Name": "",
+            "RemoveBuildArguments": 
+            {
+                "azslc": ["--strip-unused-srgs"]
+            }
+        },
+        {
+            "Name": "NoMSAA",
+            "AddBuildArguments": {
+                "azslc": ["--no-ms"]
+            },
+            "RemoveBuildArguments" :
+            {
+                "azslc": ["--strip-unused-srgs"]
+            }
+        }
```

Comment:
This super variant is only needed if the shader is sampling MSAA texture. Otherwise it will build a whole new byte code version which will be an exact duplicate.

Reply:
Removed "NoMSAA" supervariant